### PR TITLE
Feature/profile redesign see all

### PR DESCRIPTION
**Pull request by aditilamkhede**
### What does this pull request aim to achieve? 💯

_Please be concise and link any related issue(s):_

- [ ] I have checked that no one else is working on similar changes.
- [ ] I have read the latest [**README**](README.md) and understand the development workflow.
- [ ] The name of this branch is: **`feature/<branch_name>`** for a regular feature, **`hotfix/<branch_name>`** for a hotfix, and **`service/<branch_name>`** for a microservice, where `<branch_name>` briefly describes the issue(s) resolved and is prefixed with the issue number(s) (e.g. `1127-1130-update-git-branching-model`). The branch is created in a cloned version of this repo, **not a forked repo**.
- [ ] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [ ] There are no merge conflicts.
- [ ] There are no console warnings and errors.
- [ ] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".
- [ ] The title describes the issue(s) being addressed using format: **`<issue_numbers> - <brief_issue_description>`** (e.g. `#1127/#1130 - Update Git Branching Model`)

### CODE REVIEW

File: client/src/components/Profile/ProfileDesktop.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+  isLoading,
+  isItemLoaded,
+  itemCount,
+  loadMore,
+  totalPostCount,
+  isMobile,
+}) => {
+  const { t } = useTranslation();
+
+  const defaultState = useCallback(() => {
```
Comment (linadinh2602) [inline client/src/components/Profile/ProfileDesktop.js:66]: Why do we need useCallback?

File: client/src/components/Profile/ProfileDesktop.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+  });
+  const [childTab, setChildTab] = useState(defaultState);
+
+  const handleTabChange = (e) => {
+    // console.log("tab Change", e.key);
+    setChildTab(e.key);
+    setInternalTab(e.key);
+  };
+
+  useEffect(() => {
```
Comment (linadinh2602) [inline client/src/components/Profile/ProfileDesktop.js:79]: why do we need to useEffect?

File: client/src/components/Profile/ProfileDesktop.js [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+        ))}
+      </ChildMenuWrapper>
+      <WhiteSpace size={"lg"}></WhiteSpace>
+      <Activity
+        postDispatch={postDispatch}
+        filteredPosts={profilePost}
+        user={user}
+        postDelete={postDelete}
+        handlePostDelete={handlePostDelete}
+        // // handleEditPost={handleEditPost}
```
Comment (linadinh2602) [inline client/src/components/Profile/ProfileDesktop.js:110]: is this a comment out? do you mean to delete it?

File: client/src/components/Profile/ProfileDesktop.js [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+        deleteModalVisibility={deleteModalVisibility}
+        handleCancelPostDelete={handleCancelPostDelete}
+        loadNextPage={loadNextPage}
+        isNextPageLoading={isLoading}
+        itemCount={itemCount}
+        isItemLoaded={isItemLoaded}
+        hasNextPage={loadMore}
+        totalPostCount={totalPostCount}
+      />
+      {/* {postsError && (
```
Comment (linadinh2602) [inline client/src/components/Profile/ProfileDesktop.js:120]: is this a comment out? do you mean to delete it?

**Review by linadinh2602 — APPROVED** [review]
Just some comment questions but feel free to merge

File: client/src/components/Profile/ProfileDesktop.js (hunk 1 above)
Comment (aditilamkhede) [reply client/src/components/Profile/ProfileDesktop.js:66]: hmmm, I didn't add that. @clubkemp did you? If we don't see any use here, we can get rid of it.

File: client/src/components/Profile/ProfileDesktop.js (hunk 2 above)
Comment (aditilamkhede) [reply client/src/components/Profile/ProfileDesktop.js:79]: when we change parent Tabs, we want to set first tab (child) as default active tab. and it should trigger fetch too.

File: client/src/components/Profile/ProfileDesktop.js (hunk 3 above)
Comment (aditilamkhede) [reply client/src/components/Profile/ProfileDesktop.js:110]: @linadinh2602 good question. I am very amazed here, we need this edit call. Though right now it's commented still edit works. So, can we check this? if you or @clubkemp can any of you investigate this? I didn't get time to look into it more. I forgot about this to mention. I am glad you reminded me.

File: client/src/components/Profile/ProfileDesktop.js (hunk 4 above)
Comment (aditilamkhede) [reply client/src/components/Profile/ProfileDesktop.js:120]: yes, we can get rid of it.

File: client/src/components/Profile/ProfileDesktop.js (hunk 3 above)
Comment (linadinh2602) [reply client/src/components/Profile/ProfileDesktop.js:110]: I can try to take a look while working on my branch

**Pull request merged**